binding/python/test_porcupine.py

```diff
@@ -16,50 +16,42 @@
 from porcupine import Porcupine
 from util import *
 
+def _append_language(dir, language):
+    if language == 'en':
+        return dir
+    return dir + '_' + language
 
-class PorcupineTestCase(unittest.TestCase):
-    def test_process(self):
-        porcupine = Porcupine(
-            access_key=sys.argv[1],
-            library_path=pv_library_path('../..'),
-            model_path=pv_model_path('../../'),
-            keyword_paths=[pv_keyword_paths('../..')['porcupine']],
-            sensitivities=[0.5])
-
-        audio, sample_rate = soundfile.read(
-            os.path.join(os.path.dirname(__file__), '../../resources/audio_samples/porcupine.wav'),
-            dtype='int16')
-        assert sample_rate == porcupine.sample_rate
+def pv_model_path_by_language(relative, language):
+    model_path_subdir = _append_language('lib/common/porcupine_params', language) + '.pv'
+    return os.path.join(os.path.dirname(__file__), relative, model_path_subdir)
 
-        num_frames = len(audio) // porcupine.frame_length
-        results = []
-        for i in range(num_frames):
-            frame = audio[i * porcupine.frame_length:(i + 1) * porcupine.frame_length]
-            keyword_index = porcupine.process(frame)
-            if keyword_index >= 0:
-                results.append(keyword_index)
+def pv_keyword_paths_by_language(relative, language):
```

Comment:
I wanted these functions to resemble their counterpart in util.py as much as possible, so if we ever changed them it would be easy to apply those changes to these new functions as well. Let me know if that makes sense or I should remove this parameter.

Reply:
ok, sounds good